lib/ex_aws/utils.ex

```diff
@@ -100,4 +100,6 @@ defmodule ExAws.Utils do
 
     greg_secs - @seconds_0_to_1970
   end
+
+  def uuid, do: DateTime.utc_now |> :erlang.phash2
```

Comment:
I think it is good enough for a simple uuid. If we were to consider improving it, then using external package like https://hex.pm/packages/uuid could be a better alternative. I didn't want to bring in a package unless really needed.